bbbeasy-frontend/src/components/Install.tsx

```diff
@@ -165,8 +165,8 @@ const Install = () => {
                 getPresetSettings();
             })
             .catch((error) => {
-                if (error.code === 'ERR_NETWORK') {
```

Comment:
@hanazarraa can we improve it with something like `error.response.status !== 200` ?